go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/r3labs/diff/v2
+module github.com/j0s/diff/v2
```

Comment:
I think you forgot to change this before opening the PR? :stuck_out_tongue:

Reply:
Oops, pushed that change after I made the PR so I could do a release from my own repo, forgot that pushes to that branch would also go into the PR :) Anyway, should be OK now!